.travis.yml

```diff
@@ -2,9 +2,8 @@ sudo: false
 cache: bundler
 language: ruby
 rvm:
-  - "1.9.3"
   - "2.0.0"
   - "2.1.0"
   - "2.2.0"
-  - jruby-19mode
-  - jruby-21mode
```

Comment:
They screwed themselves by not supporting even the good parts of ruby 2.x. :smile: